.github/workflows/maven-v3.yml

```
@@ -0,0 +1,55 @@
+  push:
+    branches: [ "v3.x" ]
+  pull_request:
+    branches: [ "v3.x" ]
+
+jobs:
+  build-vaadin14:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v4
+    - name: Set up JDK
+      uses: actions/setup-java@v3
```

actionlint (1.7.10)

the runner of "actions/setup-java@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)